applications/notes/HM.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+</head>
+<body>
+<header>
+    <section>
+        <h1> 
+            Notedly
+
+        </h1>
+    </section>
+    <hr />
+    <h2>
+        <ul>
+            <li> Home</li>
+            <li>My Notes</li>
+            <li>Favourites</li>
+            <li>New</li>
+        </ul>
+    </h2>
+    <h3>
+        Books to Read
+        <pr>
```

Comment:
The element you're looking for here is `<p>`, but it's not necessary if it's just going to include a list.